app/main.py

```diff
@@ -1,34 +1,83 @@
+from __future__ import annotations
+
+
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(
+            self,
+            row: int,
+            column: int,
+            is_alive: bool = True
```

Comment:
In case you provide a default value for an argument, the argument becomes optional and you can indicate this in the type annotation:
```suggestion
            is_alive: bool | None = True
```